src/lib/Preview.js

```diff
@@ -1253,8 +1253,8 @@ class Preview extends EventEmitter {
 
         clearTimeout(this.retryTimeout);
 
-        // Respect 'Retry-After' header if present, otherwise retry using exponential backoff
-        let timeoutMs = 2 ** this.retryCount * MS_IN_S;
+        // Respect 'Retry-After' header if present, otherwise retry full jitter
+        let timeoutMs = Math.random() * (2 ** this.retryCount * MS_IN_S);
```

Comment:
Math.random() is potentially 0, would that be a problem if we very rarely retry right away?

Reply:
Don't think so.